lib/core/utils/publish-metadata.js

```diff
@@ -7,20 +7,32 @@
  * @private
  */
 function getIncompleteReason(checkData, messages) {
+	function getDefaultMsg(messages) {
+		if (messages.incomplete && messages.incomplete.default) {
+			// fall back to the default message if no reason specified
+			return messages.incomplete.default;
+		} else {
+			// TODO: localize it
+			return 'aXe couldn\'t tell the reason. Time to break out the element inspector!';
+		}
+	}
 	if (checkData && checkData.missingData) {
-		var missingReason;
 		try {
-			missingReason = checkData.missingData[0].reason;
-		} finally {
+			var msg = messages.incomplete[checkData.missingData[0].reason];
```

Comment:
This throws an error in the wild (enough to break our extensions) but in testing it didn't for some reason, so I threw an error just to be sure.